pkg/cluster/scheduler.go

```diff
@@ -31,5 +33,8 @@ func runScheduler(ctx context.Context, dataDir string) error {
 		logger.Infof("kubernetes scheduler exited %v", command.Execute())
 	}()
 
+	time.Sleep(time.Second * 20)
+	wg.Done()
```

Comment:
I think this is meant to be in the go func above.  not sure using wait groups in synchronous code makes much sense

Reply:
oops, that sleep was debugging code, removed.
agreed on wait groups. should this be async and wgs, or all sync?  any preference?